Makefile

```diff
@@ -61,13 +61,19 @@ ifeq ($(DEEP_NET), tensorflow)
 # Torch
 else ifeq ($(DEEP_NET), torch)
 	# COMMON_FLAGS += -DUSE_TORCH
+else ifeq ($(DEEP_NET), tensorrt)
+        COMMON_FLAGS += -DUSE_TENSORRT
 # Caffe
 else
 	COMMON_FLAGS += -DUSE_CAFFE
 	LIBRARIES += caffe
 	LDFLAGS += -Wl,-rpath=$(CAFFE_DIR)/lib
 	INCLUDE_DIRS += $(CAFFE_DIR)/include
 	LIBRARY_DIRS += $(CAFFE_DIR)/lib
+        
+        ifeq ($(DEEP_NET), tensorrt)
+                COMMON_FLAGS += -DUSE_TENSORRT
+        endif
 endif
```

Comment:
If DEEP_NET is `tensorrt` then the `else` clause is never reached hence lines `73-76` are not needed. It looks like the libraries and dirs that are part of the else clause will still be needed for `tensorrt` however so that should be moved up.

Reply:
For now TensorRT is included for the main inferences on the middle of a pipeline using CAFFE, for example I use caffe blobs for input and output. I think it's line 64-65 that should be removed.